server.go

```diff
@@ -340,6 +372,168 @@ func (a *App) VerifyHandler(w http.ResponseWriter, r *http.Request) {
 	logRequest(r, 200)
 }
 
+func (a *App) LocksHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	repo := vars["repo"]
+
+	enc := json.NewEncoder(w)
+	ll := &LockList{}
+
+	w.Header().Set("Content-Type", metaMediaType)
+
+	locks, nextCursor, err := a.metaStore.FilteredLocks(repo,
+		r.FormValue("path"),
+		r.FormValue("cursor"),
+		r.FormValue("limit"))
+
+	if err != nil {
+		ll.Message = err.Error()
```

Comment:
I noticed that this branch still hits L398 below, indicating that it is a 2xx response. Is that correct?